server/backend/api/app/schemas.py

```diff
@@ -172,3 +172,28 @@ class FamForestClientGet(FamForestClientCreate):
 
     class Config:
         orm_mode = True
+
+
+class FamUserRoleXref(FamUserRoleAssignmentGet):
```

Comment:
It is confusing from the two names: "FamUserRoleXref" and "FamUserRoleAssignmentGet" , that they seem to be the same purpose....
Sorry Kevin, maybe I am confused, for the rest of the schema objects, not sure if you have time, we can discuss for my understanding how the response object would look like.